index.html

```diff
@@ -162,41 +162,35 @@ <h2>Photo Gallary</h2>
         </section>
         <section class="survey">
             <h2>Survey</h2>
-                <form>
-                    What is your name? <input type="text" name="name_feild" id="name">
-                    <br></br>
-                    What is your email? <input type="text" name="email_feild" id="email">
-                    <br></br>
-                    Which integer is best <input type="number" name="best_integer" id="integer">
-                    <br></br>
-                    Which day is best? <input type="text" name="best_day" id="day">
-                    <br></br>
-                    Which bear is best? <select name="best_bear">
-                                            <option value="Blank">  </option>
-                                            <option value="Brown">brown</option>
-                                            <option value="Black">black</option>
-                                            <option value="Care">care</option>
-                                            <option value="Panda">panda</option>
-                                            <option value="Polar">polar</option>
-                                            <option value="Teddy">teddy</option>
-                                        </select>
-                    <br></br>
-                    Do you like radio buttons? <label id="radio">
-                    <br></br>
-                    <label>
-                        <input type="radio" name="rad" value="Yes" id="radio">Yes
-                        <br></br>
-                        <input type="radio" name="rad"value="No" id="radio">No
-                        <br></br>
-                        <input type="radio" name="rad"value="?" id="radio" checked="true"> None of your business
-                    </label>
-                    <br></br>
-                    What is the meaning of life?<label>
-                    <br></br>
-                    <textarea name="life_meaning" rows="10">
-                    Enter the correct answer here
-                    </textarea>
-                    <br></br>
+                <form method="POST" action="?">
+                    <label for="name">What is your name?</label>
+                    <input type="text" name="name_field" id="name" required>
```

Comment:
Name field should be required
You had a typo in your name attribute `field` vs `feild`